keel-core/src/main/kotlin/com/netflix/spinnaker/keel/lifecycle/LifecycleMonitor.kt

```diff
@@ -61,4 +48,40 @@ abstract class LifecycleMonitor(
     log.debug("${this.javaClass.simpleName} has completed monitoring for $task")
     monitorRepository.delete(task)
   }
+
+  /**
+   * Handles persisting failures and publishing exception events.
+   *
+   * Call when there was a failure getting status.
+   */
+  fun handleFailureFetching(task: MonitoredTask) {
```

Comment:
Suggestion:
```suggestion
  fun handleFailureFetchingTaslStatus(task: MonitoredTask) {
```

Reply:
What's a `tasl`? :trollface:

Reply:
This was just a test to see who was paying attention... 😝